src/cli/commands/version.js

```diff
@@ -130,6 +131,14 @@ export async function setVersion(
     return () => Promise.resolve();
   }
 
+  function buildCommitArgs(args): Array<string> {
+    args = args || ['commit'];
+    if (!flags.commitHooks || !config.getOption('version-commit-hooks')) {
```

Comment:
I think this part would be more useful as a separate function: `isCommitHooksDisabled()`. Then you'd use it like as follows:
```
// create git commit
await spawnGit(['commit', ...(isCommitHooksDisabled ? ['-n'] : []), '-m', message], {cwd: gitRoot});
```